R/importQIIME2.R

```diff
@@ -68,57 +80,63 @@
 #' \url{https://qiime2.org}
 #'
 #' @examples
-#' featureTableFile <- system.file("extdata", "table.qza", package = "mia")
-#' taxonomyTableFile <- system.file("extdata", "taxonomy.qza", package = "mia")
-#' sampleMetaFile <- system.file("extdata", "sample-metadata.tsv", package = "mia")
-#' phyTreeFile <- system.file("extdata", "tree.qza", package = "mia")
-#' refSeqFile <- system.file("extdata", "refseq.qza", package = "mia")
+#' assay.file <- system.file("extdata", "table.qza", package = "mia")
+#' row.file <- system.file("extdata", "taxonomy.qza", package = "mia")
+#' col.file <- system.file("extdata", "sample-metadata.tsv", package = "mia")
+#' phy.tree.file <- system.file("extdata", "tree.qza", package = "mia")
```

Comment:
Can you convert all phy.tree.file parameter names to tree.file. That is more simple

Reply:
Otherwise, I think this naming assay.file, col.file... is good, but other opinions are very welcome

Reply:
ACK.